.gitattributes

```diff
@@ -0,0 +1,10 @@
+.codeclimate.yml export-ignore
+.editorconfig export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+composer.lock export-ignore
+phpunit.xml.dist export-ignore
+services.yml export-ignore
```

Comment:
The `services.yml` is needed for the Symfony Dependency Injection component. Without it, the command line utility will be unusable when installed via Composer.